packages/webapp-alternative/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "webapp-alternative",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.16.4",
```

Comment:
Let's split dependencies and devDependencies.

Reply:
Yep, of course. It's original code from create-react-app and it's quite strange that authors of create-react-app added test libs to dependencies 🤔 I don't think they were unaware of potential issues with bundle size. I guess these libs aren't explicitly imported in code, so technically it's safe to keep them in dependencies, but in terms of separation of concern it's definitely better to move them to dev dependencies